runtime.md

```diff
@@ -67,38 +62,55 @@ Note: these operations are not specifying any command-line APIs, and the paramen
 `state <container-id>`
 
 This operation MUST generate an error if it is not provided the ID of a container.
+Attempting to query a container that does not exist MUST generate an error.
 This operation MUST return the state of a container as specified in the [State](#state) section.
-In particular, the state MUST be serialized as JSON.
+The state MUST be serialized as JSON.
 
 
-### Start
+### Create
 
-`start <container-id> <path-to-bundle>`
+`create <container-id> <path-to-bundle>`
 
 This operation MUST generate an error if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error.
-Using the data in `config.json`, that are in the bundle's directory, this operation MUST create a new container.
-This includes creating the relevant namespaces, resource limits, etc and configuring the appropriate capabilities for the container.
-A new process within the scope of the container MUST be created as specified by the `config.json` file otherwise an error MUST be generated.
+If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST not be created.
+Using the data in [`config.json`](config.md), this operation MUST create a new container.
+This includes creating, or entering, the namespaces specified in the [`config.json`](config.md), resource limits, etc. and configuring the appropriate capabilities for the container.
+If the `config.json` specifies that a PID namespace is to be created then one MUST be created, but the user-specified code within that namespace MUST NOT be run at this time.
+In some implementations this means that a temporary process is created in the PID namespace but it pauses until the `start` operation is invoked before replacing the process with the user-specified code.
+
+Any error generated during the creation of the container MUST generate an error and a new container MUST not be created.
```

Comment:
This changes the [wording][1] from #418.  I like the old wording better because for two reasons:

* Trivially, “error generated” → “MUST generate an error” sounds cyclic (although I understand you mean “forward errors to the caller”).
* Importantly, “jump to cleanup” is easier to implement than “don't create a new container”, because it covers cases where you got partway through creation before hitting an error.  If you require the runtime to perform *no* actions (vs. some actions which it then unwinds), you're back to requiring pre-create validation, and we don't want that (#404).

[1]: https://github.com/opencontainers/runtime-spec/pull/418/files#diff-b84a8d65d8ed53f4794cd2db7e8ea731R82